msyncd/synchronizer.cpp

```diff
@@ -184,7 +186,7 @@ bool Synchronizer::initialize()
         else
         {
             QObject::connect(&iSyncOnChangeScheduler, SIGNAL(syncNow(QString)),
-                             this, SLOT(startSync(QString)),
+                             this, SLOT(startScheduledSync(QString)),
```

Comment:
I consider a ScheduledSync any sync that starts without a manually request. The only difference btw a startSync and a startScheduledSync is that the startScheduledSync will check for network connections before call 'startSync'. And I believe that syncs triggered by database change should have the same policy as schedule syncs.